test/getImageMimeType.test.js

```diff
@@ -0,0 +1,5 @@
+const getImageMimeType = require("../util/getImageMimeType.js");
+
+it("Strips the extension from the filename", () => {
```

Comment:
I was confused by this descriptor. 
Maybe you can use something like `it("Uses the extension of the filename to determine mimetype")`

Reply:
(also, I would add a couple more cases, just to be complete and because it's cheap)